pkg/util/ciutil/systems.go

```diff
@@ -29,7 +31,7 @@ const (
 	Hudson                      System = "Hudson"
 	Jenkins                     System = "Jenkins"
 	MagnumCI                    System = "Magnum CI"
-	MicrosoftTFS                System = "Microsoft Team Foundation Server"
+	PulumiCI                    System = "Pulumi CI"
```

Comment:
TBH I'm not a huge fan of "Pulumi CI" as the default, catch-all. Since it might confuse some people to think that we offer some CI/CD product. Or, if we ever do offer a CI/CD product, would then require us to rename this value.

It seems like "Generic CI" would be more appropriate for the use-case? ... though since you are specifying the generic CI system _to Pulumi_, I guess prefixing the env vars with `PULUMI_` makes sense. Naming is hard.

Reply:
I actually didn't add this. I just removed MicrosoftTFS and replaced it with AzurePipelines. The reason you see a diff here is, because of the line before it being removed, causing `gofmt` to reformat the line below it.

Let me know if you still would like me to consider renaming this. However, I would like to leave it as-is since that was not the intent of this PR.

Reply:
> However, I would like to leave it as-is since that was not the intent of this PR.

👍

Reply:
Updating based on my side conversation with Chris. Turns out that I was totally wrong here. I _did_ in fact make this change myself. After speaking with Chris, it made sense to just change this to "Generic CI" as a fallback case when we don't have the code to detect a particular CI system's env vars, a user could set the relevant env vars for this "Generic CI" and still allow the Pulumi CLI to pick-up the relevant metadata about the build.